backend/src/main/java/gov/cdc/usds/simplereport/api/testresult/TestResultMutationResolver.java

```diff
@@ -27,15 +27,19 @@ public class TestResultMutationResolver {
   @MutationMapping
   @AuthorizationConfiguration.RequireGlobalAdminUser
   public boolean resendToReportStream(
-      @Argument List<UUID> testEventIds, @Argument boolean fhirOnly) {
+      @Argument List<UUID> testEventIds, @Argument boolean fhirOnly, @Argument boolean covidOnly) {
     testEventRepository
         .findAllByInternalIdIn(testEventIds)
         .forEach(
             testEvent -> {
-              if (!fhirOnly) {
+              if (fhirOnly) {
```

Comment:
If someone puts `true` for `fhirOnly` and `true` for `covidOnly` thinking that means it would resent to both pipelines, it would only send to the fhir pipeline only in this case

Reply:
Do you think changing the argument to be an explicit string argument of either `fhir` or `covid` would be better? The alternative for keeping two distinct boolean args would look like this which is okay but maybe too many conditionals?

```
              if (fhirOnly) {
                fhirReportingService.report(testEvent);
              }
              if (covidOnly) {
                testEventReportingService.report(testEvent);
              }
              if ((fhirOnly && covidOnly) || (!fhirOnly && !covidOnly)) {
                testEventReportingService.report(testEvent);
                fhirReportingService.report(testEvent);
              }
```